### ISSUE TITLE
[UX] Pages should not allow scheduling by default

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3795]
In the standard install profile, the `Show option for scheduling ` setting for Pages should be disabled.

**Comment by klonos** [comment]
I've seen use cases with customers where they'd want the option to schedule publishing of pages and other content types was required. Any specific reason why should not allow this by default for all content types?

**Comment by jenlampton** [comment]
> Any specific reason why we should not allow this by default for all content types?

Because it's uncommon, and not hard to enable when people need it :)

**Comment by klonos** [comment]
Fair enough. Adding the "needs metrics" tag; not for the decision to implement this; just because it will be a good candidate to test #285

**Comment by mmachinen** [comment]
@stpaultim I would like to work on this issue.

**Comment by klonos** [comment]
Thank you for working on this @mmachinen 🙏🏼 ...there are test failures that need to be fixed, so I'm setting this back to "needs work". It seems to me that things should be fixed in the `testPagePreview()` function/method, since it seems to be using the `page` content type in order to test scheduling among other things. I haven't tested, but changing things like these should do it (there might be other additional bits here and there though):
- `$this->backdropPost('admin/structure/types/manage/page', $edit, t('Save content type'));`
- `$this->backdropGet('node/add/page');`
- `$this->backdropPost('node/add/page', $edit, t('Preview'));`

PS: Once #5236 is merged, you should rebase your PR branch (let us know if you need help with that), which should help to display these failures in a better format 😉

**Comment by jenlampton** [comment]
I do govt work too, and they schedule lots of types of content, just as you mentioned, but never pages. It's easy to find this option when creating a new content type. I still feel strongly that we don't need it on Page.

This is one of the places where Pages differ from Posts, and I think that differentiation will also be helpful for WP people coming to Backdrop, to help them understand why we have content types.

**Comment by stpaultim** [comment]
I certainly don't feel strongly about this. But, I can see where this is not a necessary default option (scheduling pages). In terms of discoverability, having it enabled on posts helps with this and posts are MUCH more likely to be scheduled. 

It seems that "cards" are already off by default. 

I support this change. We have a PR, so I will add this to the milestone and mark WFM.

**Comment by quicksketch** [comment]
I'm on the fence about whether this is a positive change, but I can accept the mentality that it's always easier to add something than remove it (even if it's just a default setting). So let's try this change.

I merged https://github.com/backdrop/backdrop/pull/5000 into 1.x and 1.33.x. Thank you @NormPlum, @mmachinen, @jenlampton, @klonos, and @stpaultim for your work on this issue.
